### ISSUE TITLE
testnet mo-cyst-fail from growing %hall-prize to %json

### ISSUE THREAD

**Issue by loganallenc** [urbit/arvo#770]
After booting my Urbit this morning I began receiving these issues.

```
[ %mo-cyst-fail
  ! /~locnyl-dacdel/home/~2018.7.23..18.37.49..3649/sys/vane/ford:<[1.306 24].[1.306 52]>
  ! nest-fail
  !   ford: keel
        -mute.+6
      [   #t/
          piz
        ?(
          {$public cis/nlr({hos/@p nom/@tas})}
          { $burden
              sos
            nlr(
              { p/@tas
                  q
                { gaz/?(^#23.{i/{aut/@p uid/@uvH aud/nlr({hos/@p nom/@tas}) wen/@da sep/#5} t/?(#23 $~)} $~)
                    cos
                  { loc/{src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$ud @ud}) tal/?({$~ u/?({$da @da} {$ud @ud})} $~)}} $~)}) cap/@t tag/nlr(@ta) fit/{cas/?($.n $.y) utf/?($.n $.y)} con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}}
                      rem
                    nlr(
                      { p/{hos/@p nom/@tas}
                        q/{src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$ud @ud}) tal/?({$~ u/?({$da @da} {$ud @ud})} $~)}} $~)}) cap/@t tag/nlr(@ta) fit/{cas/?($.n $.y) utf/?($.n $.y)} con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}}
                      }
                    )
                  }
                    pes
                  { loc/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})
                    rem/nlr({p/{hos/@p nom/@tas} q/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})})
                  }
                }
              }
            )
          }
          { $peers
              pes
            nlr(
              { p/@p
                  q
                it(
                  ?(
                    {$client $~}
                    {$public $~}
                    {$circle nom/@tas wer/?({$~ u/{hos/@p nom/@tas}} $~) wat/nlr(?($config-l $grams $group-l $config-r $group-r)) ran/?({$~ u/{hed/?({$da @da} {$ud @ud}) tal/?({$~ u/?({$da @da} {$ud @ud})} $~)}} $~)}
                    {$burden who/@p}
                    {$peers nom/@tas}
                    {$report $~}
                    {$circles who/@p}
                  )
                )
              }
            )
          }
          {$client gys/nlr({p/@t q/nlr(nlr({hos/@p nom/@tas}))}) nis/nlr({p/@p q/@t})}
          {$report $~}
          { $circle
              nes
            ?(  ^#10
              {   i
                { num/@ud
                    gam
                  { aut/@p
                    uid/@uvH
                    aud/nlr({hos/@p nom/@tas})
                    wen/@da
                      sep
                      ^#5
                    ?(
                      {$url url/{p/{p/{p/?($.n $.y) q/?({$~ u/@ud} $~) r/?({$.n p/@if} {$.y p/*''})} q/{p/?({$~ u/@ta} $~) q/*''} r/it({p/@t q/@t})} q/?({$~ u/@t} $~)}}
                      {$fat tac/^#6.?({$tank it(?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#4} {$palm p/{p/"" q/"" r/"" s/""} q/#4}))} {$text *''} {$name nom/@t tac/#6}) sep/#5}
                      {$app app/@tas sep/#5}
                      {$lin pat/?($.n $.y) msg/@t}
                      {$ire top/@uvH sep/#5}
                      {$inv inv/?($.n $.y) cir/{hos/@p nom/@tas}}
                      {$exp exp/@t res/it(?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#4} {$palm p/{p/"" q/"" r/"" s/""} q/^#4.?({i/?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#4} {$palm p/{p/"" q/"" r/"" s/""} q/#4}) t/#4} $~)}))}
                    )
                  }
                }
                t/?(#10 $~)
              }
              $~
            )
              cos
            { loc/{src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$ud @ud}) tal/?({$~ u/?({$da @da} {$ud @ud})} $~)}} $~)}) cap/@t tag/nlr(@ta) fit/{cas/?($.n $.y) utf/?($.n $.y)} con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}}
                rem
              nlr(
                { p/{hos/@p nom/@tas}
                  q/{src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$ud @ud}) tal/?({$~ u/?({$da @da} {$ud @ud})} $~)}} $~)}) cap/@t tag/nlr(@ta) fit/{cas/?($.n $.y) utf/?($.n $.y)} con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}}
                }
              )
            }
              pes
            { loc/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})
              rem/nlr({p/{hos/@p nom/@tas} q/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})})
            }
          }
          {$circles cis/nlr(@tas)}
        )
          #t/
        ?(
          { $peers
              pes
            nlr(
              { p/@p
                  q
                it(
                  ?(
                    {$client $~}
                    {$public $~}
                    {$circle nom/@tas wer/?({$~ u/{hos/@p nom/@tas}} $~) wat/nlr(?($config-l $grams $group-l $config-r $group-r)) ran/?({$~ u/{hed/?({$da @da} {$sd @sd} {$ud @ud}) tal/?({$~ u/?({$da @da} {$sd @sd} {$ud @ud})} $~)}} $~)}
                    {$burden who/@p}
                    {$peers nom/@tas}
                    {$report $~}
                    {$circles who/@p}
                  )
                )
              }
            )
          }
          { $circle
            nes/?(^#21.{i/{num/@ud gam/{aut/@p uid/@uvH aud/nlr({hos/@p nom/@tas}) wen/@da sep/#10}} t/?(#21 $~)} $~)
              cos
            {   loc
              {src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$sd @sd} {$ud @ud}) tal/?({$~ u/?({$da @da} {$sd @sd} {$ud @ud})} $~)}} $~)}) cap/@t tag/nlr(@ta) fit/{cas/?($.n $.y) utf/?($.n $.y)} con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}}
                rem
              nlr(
                { p/{hos/@p nom/@tas}
                    q
                  { src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$sd @sd} {$ud @ud}) tal/?({$~ u/?({$da @da} {$sd @sd} {$ud @ud})} $~)}} $~)})
                    cap/@t
                    tag/nlr(@ta)
                    fit/{cas/?($.n $.y) utf/?($.n $.y)}
                    con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}
                  }
                }
              )
            }
              pes
            { loc/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})
              rem/nlr({p/{hos/@p nom/@tas} q/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})})
            }
          }
          { $burden
              sos
            nlr(
              { p/@tas
                  q
                {   gaz
                  ?(  ^#13
                    {   i
                      { aut/@p
                        uid/@uvH
                        aud/nlr({hos/@p nom/@tas})
                        wen/@da
                          sep
                          ^#10
                        ?(
                          {$ire top/@uvH sep/#10}
                          {$url url/{p/{p/{p/?($.n $.y) q/?({$~ u/@ud} $~) r/?({$.n p/@if} {$.y p/*''})} q/{p/?({$~ u/@ta} $~) q/*''} r/it({p/@t q/@t})} q/?({$~ u/@t} $~)}}
                          {$lin pat/?($.n $.y) msg/@t}
                          {$app app/@tas sep/#10}
                          {$fat tac/^#8.?({$tank it(?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#7} {$palm p/{p/"" q/"" r/"" s/""} q/#7}))} {$text *''} {$name nom/@t tac/#8}) sep/#10}
                          {$inv inv/?($.n $.y) cir/{hos/@p nom/@tas}}
                          {$exp exp/@t res/it(?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#7} {$palm p/{p/"" q/"" r/"" s/""} q/^#7.?({i/?({$leaf p/""} {$rose p/{p/"" q/"" r/""} q/#7} {$palm p/{p/"" q/"" r/"" s/""} q/#7}) t/#7} $~)}))}
                        )
                      }
                      t/?(#13 $~)
                    }
                    $~
                  )
                    cos
                  {   loc
                    { src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$sd @sd} {$ud @ud}) tal/?({$~ u/?({$da @da} {$sd @sd} {$ud @ud})} $~)}} $~)})
                      cap/@t
                      tag/nlr(@ta)
                      fit/{cas/?($.n $.y) utf/?($.n $.y)}
                      con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}
                    }
                      rem
                    nlr(
                      { p/{hos/@p nom/@tas}
                          q
                        { src/nlr({cir/{hos/@p nom/@tas} ran/?({$~ u/{hed/?({$da @da} {$sd @sd} {$ud @ud}) tal/?({$~ u/?({$da @da} {$sd @sd} {$ud @ud})} $~)}} $~)})
                          cap/@t
                          tag/nlr(@ta)
                          fit/{cas/?($.n $.y) utf/?($.n $.y)}
                          con/{sec/?($journal $channel $village $mailbox) sis/nlr(@p)}
                        }
                      }
                    )
                  }
                    pes
                  { loc/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})
                    rem/nlr({p/{hos/@p nom/@tas} q/nlr({p/@p q/{pec/?($gone $hear $idle $talk) man/{han/?({$~ u/@t} $~) tru/?({$~ u/{fir/@t mid/?({$~ u/@t} $~) las/@t}} $~)}}})})
                  }
                }
              }
            )
          }
          {$client gys/nlr({p/@t q/nlr(nlr({hos/@p nom/@tas}))}) nis/nlr({p/@p q/@t})}
          {$report $~}
          {$circles cis/nlr(@tas)}
          {$public cis/nlr({hos/@p nom/@tas})}
        )
      ]
  ! ford: grow %hall-prize to %json
  ! ford: casting %hall-prize to %json
  ! ford: cast %json
]
[%quit %of p='ravper-sompes-bosfun-siclex' q=[p=~.~locnyl-dacdel q=%hall r=%mess s=/public]]
```

**Comment by loganallenc** [comment]
@vvisigoth

**Comment by vvisigoth** [comment]
Said this to Logan on chat, but copying here:

The appears to be due to a previous commit where we reintroduced -x message ranges. I'd guess a mar didn't get checked in, but I'll double check.

**Comment by Fang-** [comment]
> The appears to be due to a previous commit where we reintroduced -x message ranges. I'd guess a mar didn't get checked in, but I'll double check.

Oh yeah, I forgot to tell you, this (rough upgrade) is the exact reason that change got reverted (on livenet) in the first place. It causes issues when ships with different hall versions try to communicate.  
This in particular looks like a mark issue, yes. Thought that came with the PR I pointed you toward though?

(also idk about assigning the "priority high" label to "testnet" issues...)
